backend/src/game/game-state.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { CheckResult, GameState, Winner } from './game-state';
```

Comment:
```suggestion
import { GameState } from './game-state';
```
The other exported members do not exist.